src/ActorManager.cpp

```
@@ -382,88 +390,79 @@ namespace hdt
+
+			// we want first the smallest angle (so the highest cosinus), and the smallest distance,
+			// so we want the smallest distance / cosinus.
+			// cl = cosinus * distance, dl = distance² => distance / cosinus = dl/cl
+			// So we want dl/cl < dr/cr.
+			// Moreover, this test manages the case where one of the skeletons is behind the camera and the other in front;
+			// the one behind the camera is last (the one with cos(angle) = cr < 0).
+			return dl * cr < dr * cl;
 		});
 
 		// We set which skeletons are active and we count them.
+		const auto world = SkyrimPhysicsWorld::get();
```

Clang (14.0.6)

[warning] 'const auto world' can be declared as 'auto *const world'

(readability-qualified-auto)